pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -35,6 +35,8 @@
     <ExcludeFromClosure Include="System.Net" />
     <ExcludeFromClosure Include="System.Transactions" />
 
+    <ExcludeFromDuplicateTypes Include="System.Private.Reflection.Metadata.Ecma335" />
```

Comment:
Yes, a different name is intentional - SRM has many more types, dependency on SCI. I think we should make all types in this assembly internal though.